.expeditor/generate-automate-api-docs.sh

```diff
@@ -10,10 +10,12 @@ git checkout -b "$branch"
 
 pushd /go/src/github.com/chef/automate/components/automate-chef-io
   make sync_swagger_files
+  make generate_swagger
```

Comment:
For building the combined swagger file there were two supported paths: in the dev environment and during building of the habitat package. For the dev env, it was assumed that the developer was installing the deps and had the correct versions laid out in the README. For habitat, the dep information is in the `plan.sh`. Putting this command here pushes those dependency requirements onto expeditor, where I'm not sure how to specify/install the dependencies needed.

Reply:
@phiggins We should be OK as all the of the deps listed in the `plan.sh` are already installed in the `chefes/releng-base` Docker image which is used when Expeditor executes scripts like this:
https://github.com/chef/release-engineering/blob/master/components/dockerfiles/chefes/releng-base/Dockerfile

Reply:
Nice! This should probably be smarter about the version of `go-swagger` because I believe this is already out of sync: https://github.com/chef/release-engineering/blob/master/components/dockerfiles/chefes/releng-base/Dockerfile#L243

Reply:
What version should we be using? It looks like `v0.25.0` is the latest:
https://github.com/go-swagger/go-swagger/releases/tag/v0.25.0

Reply:
`0.20.1` is what I have in the README, not sure if that's relevant anymore: https://github.com/chef/automate/blob/master/components/automate-chef-io/README.md#one-time-install

There were some breaking changes across versions when I was doing the initial API docs work.

Reply:
Ok the updated Docker image has been fully deployed!
https://github.com/chef/release-engineering/commit/945f7112a5549129e11bd1574313d2a8fdd1f072#diff-a7ba0be291cfb9a325117dbd2cccfc68